### DISCUSSION TITLE
Dashboard Example

### DISCUSSION THREAD

**Discussion by BinomeDeNewton** [shadcn-ui/ui#632] [Q&A]
Hi, I would like to open the Dashboard example in my project so I installed all the packages that are required. When I `npx next dev`, I don't know why it compiled but it shows a very weird page like any templates were installed :
[Dashboard.pdf](https://github.com/shadcn/ui/files/11832379/Dashboard.pdf)

Can someone help me to fully installed the dashboard, please?

**Comment by dan5py** [comment]
Did you clone the whole repository? @BinomeDeNewton

**Reply by BinomeDeNewton** [reply]
I just copied the dashboard folder and included it in my project. I then followed all the steps presented on the site for installing the packages and structuring the project folder. Finally, I ran the project with `npx next dev` until it compiled, downloaded all the necessary components, and changed all the `PATH` so that it suited my project, but the result wasn't very pretty, as if it were an unstyled `HTML` page.

**Reply by BinomeDeNewton** [reply]
I can Gist some files if you want
